src/libs/elektra/keyset.c

```diff
@@ -2379,6 +2400,10 @@ int ksClose (KeySet * ks)
 
 	ks->size = 0;
 
+#ifdef ELEKTRA_ENABLE_OPTIMIZATIONS
+	opmphmDel (ks->opmphm);
```

Comment:
Better to check for null pointer in ks->opmphm. (opmphmDel has a non-documented precondition)